### PR TITLE
Add support for Electronut Labs Papyr

### PR DESCRIPTION
**Pull request by ntavish**
Circuitpython port here: https://github.com/adafruit/circuitpython/pull/1798

### CODE REVIEW

**Comment by ntavish** [comment]
Papyr's current draw sometimes spikes over 100mA, so there is a change of brownout/reset. Circuitpython was rebooting to safe mode if pins aren't set 'correctly' (seen this once or twice). That's why I included  `boards.h` inside `usb_desc.h`: to increase requested power, and to override PID. Let me know if this is not the preferred approach. I can in a separate PR also cleanup other boards if information about VID/PID is available.

**Comment by hathach** [comment]
@ntavish does your board draw more than 100 mA in bootloader mode or application mode ? Though I agree we can allow `board` to overwrite several value of USB.

File: src/usb/usb_desc.c [hunk 1]
```diff
@@ -79,30 +84,33 @@ enum {
 // STRING DESCRIPTORS
 //--------------------------------------------------------------------+
 
+#ifndef DEVICE_STRING_DESCRIPTORS
```
Comment (hathach) [inline src/usb/usb_desc.c]: should be name `USB_STRING_DESCRIPTORS` instead

**Comment by ntavish** [comment]
Oh you're right, this needs to be application mode :man_facepalming:. My excuse is that I was doing it at 3AM. The other modifications to the USB descriptors, the VID:PID for example are still required though.

**Pull request merged**